Task50/Task50/AlertsTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Threading;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Task50
+{
+    public class AlertsTests
+    {
+        private static string URL = "https://www.seleniumeasy.com/test/javascript-alert-box-demo.html";
+        private IWebDriver driver;
+        
+        [SetUp]
+        public void SetUp()
+        {
+            driver = new ChromeDriver();
+            driver.Url = URL;
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5); 
+        }
+
+        [Test]
+        public void JavaScriptAlertBoxClickedOKTest()
+        {
+            driver.FindElement(By.XPath("//button[@onclick='myAlertFunction()']")).Click();
+            var alert = driver.SwitchTo().Alert();
+            Thread.Sleep(3000);
+            var alertText = alert.Text;
+            alert.Dismiss();
+            Assert.AreEqual(alertText, "I am an alert box!");
```

Comment:
Add the third parameter to AreEqual method for getting more detailed information in case if test fail at this step